pages/apply/overview.tsx

```diff
@@ -9,11 +9,10 @@ import {
   SummaryListRow as Row,
 } from '../../components/summary-list';
 import ApplicantSummary from '../../components/application/ApplicantSummary';
-import { Applicant } from '../../domain/HousingApi';
+import { selectApplicantsMemorised } from '../../lib/store/applicant';
```

Comment:
What does selectApplicantsMemorised do? Is it about using the saved applicant in some way?

Reply:
Have a look at https://github.com/LBHackney-IT/lbh-housing-register/pull/437

Basically will cache and avoid re-renders which was the (one of many) issue(s) we have been having.